src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -24,13 +24,13 @@ def map_id_bus(scenario):
             "Diele": "177829920",
             "Dörpen/West": "142487746",
             "Emden/Borßum": "34835258",
-            "Emden/Ost": "34835258",
+            "Emden/Ost": "177829920",
             "Hagermarsch": "79316833",
             "Hanekenfähr": "61918154",
             "Inhausen": "29420322",
             "Unterweser": "32076853",
             "Wehrendorf": "33411203",
-            "Wilhelmshaven 2": "23744346",
+            "Wilhelmshaven 2": "23837631",
```

Comment:
Why did you change these osm_ids?

Reply:
Since we changed the OSM year, some ids changed.

Reply:
So we also use 2019 osm data for the future scenarios? 
I would somehow like to be compatible to the newer osm version, at least if we already collected the osm_ids once. 
Could you probably check the year of the used osm data and set different osm_ids for that? Sth. similar like here: https://github.com/openego/powerd-data/blob/12db0fecc9c12f92646c02780eeb1511e474c2ca/src/egon/data/datasets/osmtgmod/__init__.py#L184

Reply:
According to our conversation, I applied the changes in regard to this comments here d2d815b63fca3773328cdaaa946622bff921f0e7